building/build_options.adoc

```diff
@@ -0,0 +1,67 @@
+# Options for building JBoss Tools
+
+## Maven options
+
+Here are the options we use for JBoss Tools builds:
```

Comment:
changed to:

"Here are the options we recommend to learn (& use as needed) when running JBoss Tools builds:"